ormar/models/model.py

```diff
@@ -220,6 +223,7 @@ async def update(self: T, _columns: List[str] = None, **kwargs: Any) -> T:
         :rtype: Model
         """
         if kwargs:
+            kwargs = self.populate_onupdate_value(kwargs)
```

Comment:
That means we always overwrite the field value with onupdate value.

If someone specifies the value for a column explicitly in an update it should have priority over the onupdate value (so you can explicitly overwrite it if you want).

You expressed the same idea in docs above where you wrote: `if you don't update the field which has the onupdate option, its value will be changed from onupdate definition`. But with the current implementation, it's always overwritten, even if you do update the field explicitly.